gonzo/tasks/gonzo.py

```diff
@@ -59,9 +34,8 @@ def group(*env_type_pairs):
         # env_type_pair is e.g. produiction-platform-app
         # we want production, and platform-app
         environment, server_type = env_type_pair.split("-", 1)
-
-        instances = cloud.get_instance_by_tags(
-            environment=environment, server_type=server_type)
-        env.hosts.extend([get_hostname(inst) for inst in instances])
+        instances = cloud.list_instances_by_type(
+            environment=environment, instance_type=server_type)
```

Comment:
could we be consistent (here and the rest of the code base), calling this `instance_type` _or) `server_type`